### DISCUSSION TITLE
Feature request - Training Scripts

### DISCUSSION THREAD

**Discussion by dpny518** [snakers4/silero-models#12] [Ideas]
## 🚀 Feature
Training procedure using kaldi's data/train data/test input with wav.scp, utt2spk, text inputs or deepspeech's audiofile text 

## Motivation

Want to train our own models

## Pitch
Useful for the community and can utilize the first part of the data prep from all the kaldi recipes
## Additional context
None

**Comment by snakers4** [comment]
Hi,

We decided against sharing our training scripts for the time being for a number of reasons.

If in future we will share them, most likely we will rely on plain manifest format, i.e. a plain file with audio and transcript paths.

As for widely circulated recipes and such (usually for English) they usually rely either (i) on small academic datasets (ii) datasets inaccessible for public (like switchboard or fisher), which limits their usability in real life.

As I mentioned in one of the issues - if you would like to have a low resource language added  or a language where we have no plans in having an EE model (Chinese / Arabic / Tagalog / Hindi) - please ping me in private - we may just add it as a service to community.

I think I need to provide some write up on this in wiki.

**Comment by snakers4** [comment]
https://github.com/snakers4/silero-models/wiki/Adding-New-Languages
